### DISCUSSION TITLE
如何在双拼输入下使用“快捷短语”？

### DISCUSSION THREAD

**Discussion by kuqquk** [fcitx5-android/fcitx5-android#327] [Q&A | 问答] [answered]
在设置中有一个"Use V to trigger quickphrase"的选项，但输入大写V后并不能触发内置的快捷短语。

**Comment by rocka** [comment, marked as answer]
一般双拼都会占用 v 键作为声母，所以这项设置只对全拼有用。可以通过长按逗号键，之后右划，来触发快速输入。

https://fcitx5-android.github.io/faq/#emoji-unicode
